src/renderer/components/SummaryPage.vue

```diff
@@ -22,6 +22,7 @@
 
     <el-row class="summary-page__row">
       <el-card>
+        <!-- TODO: maybe this can be replaced with <transactions> -->
```

Comment:
Once I thought this table can be replaced with `<transactions>` component, but I left it as is because this table seems to contain transactions of many wallets rather than of one wallet. I think it needs to modify when connecting API. 🤔

Reply:
I guess it is possible to pass some prop (e.g. a column names array) to modify columns. :) On the other hand, what I wonder is below.

Firstly, I think this table on a summary page contains mixed transactions of many wallets (I'm not sure this expression is correct.)

With that in mind, there are two transactions tables in the app, on a summary page and on a wallet page, which look similar but have two slightly different functionalities (I think):

1. to display transactions data associated with a wallet.
2. to display transactions data associated with an account (?) (i.e. of many wallets)

This line is the latter case. In the case, we cannot specify a wallet ID of transactions to be displayed, so we have 2 choices:

* A. to pass transactions data to the table beforehand somehow (e.g. the parent component fetches the data beforehand and passes it to the table)
  * `<transactions :data="transactionToDisplay" :columns="..." />`
* B. to have `<transaction>` obtain ALL transactions associated with the account by passing nothing.
  * `<transactions :walletId="..." :columns="..." />` displays transactions associated with an wallet
  * `<transactions :columns="..." />` displays all transactions

Is my understanding correct? What do you think? 🤔

Reply:
About transactions:

Maybe for this case we won't be THAT general (I mean with array of columns, etc). and just do something like "currency" and that's all.

About how we will fetch data:
I think that we should NOT fetch data in our "transaction" component, since it should be dumb component and we should just pass data there with a prop. I think that parent component should fetch data and pass it to the bottom to the transactions :)

Reply:
Thank you :) I made two changes:

1. pass transactions data with a prop like `<transactions :transactions="transactions" />`
1. toggle a currency column with a "currency" prop like `<transactions :transactions="transactions" currency />`